### PR TITLE
App component

### PR DESCRIPTION
**Pull request by jereleao**

### CODE REVIEW

**Review by jereleao — COMMENTED** [review]
test comment

**Review by jereleao — COMMENTED** [review]
Keep the mouse.html file local, no related with MLT

File: src/App.css [hunk 1]
```diff
@@ -2,32 +2,49 @@
   text-align: center;
 }
 
+.App-body {
```
Comment (jereleao) [inline src/App.css:5]: Changed App- header to App-body

File: src/App.js [hunk 2]
```diff
@@ -1,32 +1,106 @@
...
-              <input type="text" name="name" onChange={e => this.setState({ name: e.target.value })} />
-            </label>          
+          <h1>MLT Tag Designer</h1>
         </header>
-      </div>      
+        <div className="App-body">
+          <div className="disc-preview">
+            <Tag
+              className="App-logo"
+              typedName={this.state.name}
```
Comment (JerryLion91) [inline src/App.js:32]: change the props names to be more readeble

File: src/App.js [hunk 3]
```diff
@@ -1,32 +1,106 @@
...
+              fontFamily={this.state.font}
+              spaceBetween={this.state.space}
+              startPosition={this.state.start}
+            />
```
Comment (jereleao) [inline src/App.js:40]: Change the Tag properties names to be more readable

File: src/App.js [hunk 4]
```diff
@@ -1,32 +1,106 @@
...
+            <RotateViewer />
+          </div>
+          <div className="table">
+            <ul>
+              <li>{this.state.name}</li>
+              <li>{this.state.font}</li>
+              <li>{this.state.space}</li>
+              <li>{this.state.start}</li>
+            </ul>
+          </div>
```
Comment (jereleao) [inline src/App.js:50]: This table shows the values to manipulate whats going on

**Pull request closed without merging**